src/Sylius/Bundle/ApiBundle/CommandHandler/Checkout/AddressOrderHandler.php

```diff
@@ -76,8 +83,27 @@ public function __invoke(AddressOrder $addressOrder): OrderInterface
             $order->setCustomer($this->provideCustomerByEmail($addressOrder->email));
         }
 
-        $order->setBillingAddress($addressOrder->billingAddress);
-        $order->setShippingAddress($addressOrder->shippingAddress ?? clone $addressOrder->billingAddress);
+        /** @var AddressInterface $newBillingAddress */
+        $newBillingAddress = $addressOrder->billingAddress;
+        /** @var AddressInterface|null $newShippingAddress */
```

Comment:
```suggestion
        /** @var AddressInterface $newShippingAddress */
```
It cannot be `null` as there is `??` and even method call requires object not allowing to pass `null` into it.